tox.ini

```diff
@@ -10,17 +10,22 @@ commands=
   check-manifest
   isort --check --diff .
 deps =
-  -r requirements/dev.txt
+  flake8
+  black
+  check-manifest
+  isort
 skip_install = true
 
-[testenv:tests]
+[testenv:tests{,-ci}]
```

Comment:
Can you please elaborate this change?

Reply:
This makes `codecov` dependency only installed in CI, which runs the `tests-ci` job, whereas users would rather use the `tests` environment locally.